src/main/resources/dbreplica/migration/V1.0.10__configure_replication_plsql.sql

```diff
@@ -0,0 +1,183 @@
+--
+-- Copies a task record into the history table.
+--
+CREATE OR REPLACE FUNCTION cft_task_db.add_task_history(l_task cft_task_db.tasks, l_is_delete boolean)
```

Comment:
This names seems a bit misleading now since you're not just adding to the history any more... you're also upserting into the reportable_tasks table.

Reply:
As discussed with Ravi, this will be refactored RWA-2085